router/xgress_edge_tunnel/tunneler.go

```diff
@@ -70,6 +70,16 @@ func (self *tunneler) Start(notifyClose <-chan struct{}) error {
 	log := pfxlog.Logger()
 	log.WithField("mode", self.listenOptions.mode).Info("creating interceptor")
 
+	resolver, err := dns.NewResolver(self.listenOptions.resolver)
+	if err != nil {
+		pfxlog.Logger().WithError(err).Error("failed to start DNS resolver")
```

Comment:
There's an issue related to this code: https://github.com/openziti/ziti/issues/1513
Do you think you could pop in a quick fix for that here.. just assign a default resolver if the selected resolver initialization fails?